### ISSUE TITLE
"\u00b5IU/ml" not working for concentration (glucose/Bergman1981)

### ISSUE THREAD

**Issue by matthiaskoenig** [matthiaskoenig/pkdb#303]
```
backend_1        | -------------------------------------------------------------------------------
backend_1        | ERROR Internal Server Error: /api/v1/studies/7033284/
backend_1        | Traceback (most recent call last):
backend_1        |   File "/code/pkdb_app/categoricals.py", line 98, in norm_unit
backend_1        |     return self.dimension_to_n_unit[str(self.unit_dimension(unit))]
backend_1        | KeyError: '[length] ** 3 / [activity_amount] / [time]'
backend_1        | 
backend_1        | During handling of the above exception, another exception occurred:
backend_1        | 
backend_1        | Traceback (most recent call last):
backend_1        |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
backend_1        |     response = get_response(request)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
backend_1        |     response = self.process_exception_by_middleware(e, request)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
backend_1        |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
backend_1        |     return view_func(*args, **kwargs)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/viewsets.py", line 103, in view
backend_1        |     return self.dispatch(request, *args, **kwargs)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 483, in dispatch
backend_1        |     response = self.handle_exception(exc)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 443, in handle_exception
backend_1        |     self.raise_uncaught_exception(exc)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/views.py", line 480, in dispatch
backend_1        |     response = handler(request, *args, **kwargs)
backend_1        |   File "/code/pkdb_app/studies/views.py", line 110, in partial_update
backend_1        |     return super().partial_update(request, *args, **kwargs)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/mixins.py", line 84, in partial_update
backend_1        |     return self.update(request, *args, **kwargs)
backend_1        |   File "/code/pkdb_app/studies/views.py", line 114, in update
backend_1        |     return super().update(request, *args, **kwargs)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/mixins.py", line 70, in update
backend_1        |     self.perform_update(serializer)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/mixins.py", line 80, in perform_update
backend_1        |     serializer.save()
backend_1        |   File "/usr/local/lib/python3.6/site-packages/rest_framework/serializers.py", line 209, in save
backend_1        |     self.instance = self.update(self.instance, validated_data)
backend_1        |   File "/code/pkdb_app/studies/serializers.py", line 220, in update
backend_1        |     instance = self.create_relations(instance, related)
backend_1        |   File "/code/pkdb_app/studies/serializers.py", line 331, in create_relations
backend_1        |     instance = model.objects.create(study=study, **related[name])
backend_1        |   File "/code/pkdb_app/interventions/managers.py", line 73, in create
backend_1        |     timecourse_ex_instance = outputset.timecourse_exs.create(**timecourse_ex)
backend_1        |   File "/usr/local/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 651, in create
backend_1        |     return super(RelatedManager, self.db_manager(db)).create(**kwargs)
backend_1        |   File "/code/pkdb_app/interventions/managers.py", line 177, in create
backend_1        |     create_multiple_bulk_normalized(outputs_dj, Output)
backend_1        |   File "/code/pkdb_app/utils.py", line 62, in create_multiple_bulk_normalized
backend_1        |     return model_class.objects.bulk_create([initialize_normed(notnorm_instance) for notnorm_instance in notnormalized_instances])
backend_1        |   File "/code/pkdb_app/utils.py", line 62, in <listcomp>
backend_1        |     return model_class.objects.bulk_create([initialize_normed(notnorm_instance) for notnorm_instance in notnormalized_instances])
backend_1        |   File "/code/pkdb_app/utils.py", line 68, in initialize_normed
backend_1        |     norm.normalize()
backend_1        |   File "/code/pkdb_app/behaviours.py", line 164, in normalize
backend_1        |     setattr(self, key, self.category_class_data.normalize(value, self.unit).magnitude)
backend_1        |   File "/code/pkdb_app/categoricals.py", line 110, in normalize
backend_1        |     this_norm_unit_p = self.norm_unit(unit)
backend_1        |   File "/code/pkdb_app/categoricals.py", line 100, in norm_unit
backend_1        |     raise ValueError(f"Dimension [{self.unit_dimension(unit)}] is not allowed for pktype [{self.key}]. Dimension was calculated from unit :[{unit}]")
backend_1        | ValueError: Dimension [[length] ** 3 / [activity_amount] / [time]] is not allowed for pktype [clearance]. Dimension was calculated from unit :[(1.0 liter * milligram / kilogram / microIU)/(hr)]
```

**Comment by matthiaskoenig** [comment]
Same for `glucose/Felig1969`

**Comment by janekg89** [comment]
should be solved in ab12658f871e05547487ad2bac65f6a9adc4900a
